src/main/java/seedu/trackermon/logic/parser/CliSyntax.java

```diff
@@ -10,11 +10,6 @@
     public static final Prefix PREFIX_STATUS = new Prefix("s/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_COMMENT = new Prefix("c/");
-    public static final Prefix PREFIX_SORT_NAME_ACS = new Prefix("sna/");
-    public static final Prefix PREFIX_SORT_STATUS_ACS = new Prefix("ssa/");
-    public static final Prefix PREFIX_SORT_NAME_DES = new Prefix("snd/");
-    public static final Prefix PREFIX_SORT_STATUS_DES = new Prefix("ssd/");
-    public static final Prefix PREFIX_SORT_STATUS_ORD = new Prefix("so/");
-    public static final Prefix PREFIX_RATING = new Prefix("r/");
-
+    public static final Prefix PREFIX_RATING = new Prefix("sr/");
```

Comment:
Should this be `new Prefix("r/")` instead to keep in line with your previous lines?

Reply:
ya